tests/bears/BearTest.py

```diff
@@ -129,6 +139,24 @@ def test_bad_bear(self):
         # debug message contains custom content, dont test this here
         self.queue.get()
 
+    def test_BadLocalBear(self):
+        self.uut = BadLocalBear(self.settings, self.queue)
```

Comment:
why not giving directly `LocalBear` with a comment that because of no `run()` implementation it fails?